src/poliastro/constants.py

```diff
@@ -286,8 +286,8 @@
 R_earth = Constant(
     "R_earth",
     "Earth equatorial radius",
-    6.3781366e6,
-    "m",
+    6.3781366e3,
```

Comment:
Can we remove this unrelated change?

Reply:
Also, remember we still have https://github.com/poliastro/poliastro/issues/731

Reply:
Sure! Will open a pull request for this new change :smile: